lib/aescrypt.rb

```diff
@@ -28,19 +28,27 @@
 
 require 'openssl'
 require 'base64'
+require 'securerandom'
+
+require_relative 'migrator'
 
 module AESCrypt
-  def self.encrypt(message, password)
-    Base64.encode64(self.encrypt_data(message.to_s.strip, self.key_digest(password), nil, "AES-256-CBC"))
+  def self.encrypt(message, password, salt = nil, iv = nil)
+    iv ||= SecureRandom.bytes 16
```

Comment:
I don't know the `SecureRandom` implementation without checking. But I notice that in the [README.md](https://github.com/Gurpartap/aescrypt/pull/16/files#diff-04c6e90faac2675aa89e2176d2eec7d8R52) you suggest using `OpenSSL::Random.random_bytes(16)`. Based on knowing OpenSSL a bit, I'd use the latter.

Reply:
`SecureRandom.bytes` only for ruby 2.4 ?